src/Components/Open/displayFiles.ts

```diff
@@ -29,28 +28,36 @@ const displayFiles = async (
 	const appearance = await Storage.get('appearance');
 	const dirAlongsideFiles = preference?.dirAlongsideFiles ?? false;
 	const layout = (await Storage.get('layout'))?.[dir] ?? appearance?.layout ?? 'd';
-	const sort = (await Storage.get('sort'))?.[dir] ?? (await getDefaultSort(dir)) ?? 'a';
-
-	files = files.sort((a, b) => {
-		switch (sort) {
-			case 'A': // A-Z
-				return a.basename.toLowerCase() > b.basename.toLowerCase() ? 1 : -1;
-			case 'Z': // Z-A
-				return a.basename.toLowerCase() < b.basename.toLowerCase() ? 1 : -1;
-			case 'L': // Last Modified
-				return new Date(a.last_modified?.secs_since_epoch ?? a.time_deleted) < new Date(b.last_modified?.secs_since_epoch ?? b.time_deleted)
-					? 1
-					: -1;
-			case 'F': // First Modified
-				return new Date(a.last_modified?.secs_since_epoch ?? a.time_deleted) > new Date(b.last_modified?.secs_since_epoch ?? b.time_deleted)
-					? 1
-					: -1;
-			case 'S': // Size
-				return a.size > b.size ? 1 : -1;
-			case 'T':
-				return a.file_type > b.file_type ? 1 : -1;
-		}
-	});
+	const sort = (await Storage.get('sort'))?.[dir] ?? (await getDefaultSort(dir)) ?? 'A';
+	if (sort === 'A' || sort === 'Z') {
+		const compator = new Intl.Collator(undefined, { numeric: true, sensitivity: 'base' }).compare;
+		files.sort((a, b) => {
+			if (sort === 'A') {
```

Comment:
Avoid conditional statement inside sort function to enhance performance:
```javascript
if (sort === 'A') {
	files.sort((a, b) => compator(a.basename.toLowerCase(), b.basename.toLowerCase());
} else {
	files.sort((a, b) => compator(b.basename.toLowerCase(), a.basename.toLowerCase());
}
```

Reply:
Idk why there're so many tabs I'm confused :rofl:

Reply:
> Idk why there're so many tabs I'm confused 🤣 What do you think about using space?

Actually, it's intended to prettier it lol, when you code and save Xplorer code on VSCode, it'll auto format it using prettier.

Reply:
Ok just I'm more used to space. Tab is fine on VSCode as you can set its width but on Github it's not too good. Bad boy Github :rofl: